src/Symfony/Component/Validator/Constraints/EmailValidator.php

```diff
@@ -23,14 +24,20 @@
  */
 class EmailValidator extends ConstraintValidator
 {
+    private $strict;
+    private $strictValidator;
+
     /**
-     * @var bool
+     * @param bool $strict
      */
-    private $isStrict;
-
     public function __construct($strict = false)
     {
-        $this->isStrict = $strict;
+        $this->strict = $strict;
+    }
+
+    public function setStrictValidator(StrictEmailValidator $validator)
```

Comment:
Thought of it, and im not sure. I guess to avoid any BC damage :) But im fine with both...